packages/uniforms-antd/src/NumField.tsx

```diff
@@ -1,4 +1,6 @@
-import InputNumber, { InputNumberProps } from 'antd/lib/input-number';
+/// <reference types="rc-input-number" />
```

Comment:
What is it needed for?

Reply:
https://github.com/vazco/uniforms/pull/1375#discussion_r1817891304

It fixes issue with resolving package by `pnpm`.